source/ranges.tex

```diff
@@ -4475,7 +4475,8 @@
     using iterator_concept  =
       conditional_t<ForwardRange<Base>, forward_iterator_tag, input_iterator_tag>;
     using iterator_category = input_iterator_tag;
-    struct value_type;                                  // see \ref{range.split.outer.value}
+    // \ref{range.split.outer.value}, class \tcode{split_view::outer_iterator::value_type}
+    struct value_type;                                  
```

Comment:
```suggestion
    struct value_type;
```

Reply:
(FYI, This trailing whitespace is causing the Travis failure.)

Reply:
Fixed.